src/responses.ts

```diff
@@ -181,3 +181,19 @@ export interface GetStateResponse {
   block_hash: string
   id: string
 }
+
+export interface NearAccessKeyResponse {
```

Comment:
I'd call this `GetAccessKeyResponse` or `ViewAccessKeyResponse`

Reply:
Roger that!